src/driver/communication_interface.cc

```diff
@@ -54,7 +54,7 @@
 #include "robot_msgs/pause_cmd.hpp"          // for pause_cmd
 #include "robot_msgs/trigger_t.hpp"          // for trigger_t
 #include "utils/polynomial_encode_decode.h"  // for decodePiecewisePolynomial
-#include "utils/util_conv.h"                 // ConvertToLcmStatus
+#include "utils/util_conv.h"                 // ConvertToLcmIiwaStatus
```

Comment:
i thought we were deprecating IIWA-specific types and comments?

Reply:
We can't deprecate it yet. This lcm message is a superset of the IIWA status message. Now we can slowly start moving away from IIWA status